health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -151,47 +151,98 @@ const ViewHierarchy = () => {
+          }
+      
+          setDataCreateToast(false);
+          setShowToast({ label: `${t("WBH_HIERARCHY_CREATED")}`, isError: "success" });
+          return createResponse;
+        } catch (resp) {
+          setDisable(false);
+          let label = `${t("WBH_BOUNDARY_CREATION_FAIL")}: `;
+          resp?.response?.data?.Errors?.map((err, idx) => {
             if (idx === resp?.response?.data?.Errors?.length - 1) {
-                label = label + t(Digit.Utils.locale.getTransformedLocale(err?.code)) + ".";
+              label = label + t(Digit.Utils.locale.getTransformedLocale(err?.code)) + ".";
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)